scripts/checks/branch_adds_binary

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+set -e
+
+current_branch="$(git symbolic-ref --short HEAD)"
```

Comment:
Not sure if we want to test for "binary" files, or just for files bigger than say 200K...

Reply:
I think it would be best to add another test for the size on a commit. If the commit is too big that’s dubious. But I don’t want to miss "small" binary files.